pages/en/lb4/Sequence.md

```diff
@@ -100,17 +110,23 @@ class MySequence extends DefaultSequence {
 }
 ```
 
-In order for LoopBack to use your custom sequence, you must register it when constructing your `Application`:
+In order for LoopBack to use your custom sequence, you must register it on any
+applicable `Server` instances before starting your `Application`:
 
 ```js
 import {Application} from '@loopback/core';
+import {RestComponent, RestServer} from '@loopback/rest';
 
 const app = new Application({
-  sequence: MySequence
+  components: [RestComponent],
```

Comment:
This is a step backwards in terms of UX, at least IMO. Isn't it possible to configure the sequence via component-specific configuration? Something along the following lines:

```js
const app = new Application({
  components: [RestComponent],
  rest: {
    sequence: MySequence,
  }
});
```

Reply:
I think the current design supports your suggestion, so I'll change it.